corehq/apps/users/views/mobile/users.py

```diff
@@ -1685,6 +1731,60 @@ def link_connectid_user(request, domain):
         return HttpResponse()
 
 
+@require_can_edit_commcare_users
+@location_safe
+def send_connectid_invite(request, domain, user_id):
+    # Currently same as what send_confirmation_sms does
+    user = CommCareUser.get_by_user_id(user_id, domain)
+    if user.is_account_confirmed or not user.is_commcare_user() or user.domain != domain:
+        messages.error(request, "The user is already confirmed or is not a mobile user")
+        return HttpResponse(status=400)
+
+    invite_code = encrypt_account_confirmation_info(user)
```

Comment:
what does encrypting the invite code do for us here?

Reply:
I think the code 'encodes' the invitation sent time, so that before confirming it can check the invite didn't expire. This is really a soft enforcement used in the other two multi-stage user provision workflows, so I took the pattern as it is.